packages/phrases/src/locales/en/translation/demo-app.ts

```diff
@@ -1,6 +1,6 @@
 const demo_app = {
   title: "You've successfully signed in the live preview!",
-  subtitle: 'Here is your log in information:',
+  subtitle: 'Here is your user information:',
```

Comment:
It seems the `subtitle` is missing from other languages

Reply:
because i changed the content

Reply:
Ah, OK, auto fallback